kpops/components/base_components/kafka_app.py

```diff
@@ -63,23 +66,24 @@ def deploy(self, dry_run: bool):
                 )
         super().deploy(dry_run)
 
-    def destroy(self, dry_run: bool, clean: bool, delete_outputs: bool) -> None:
-        super().destroy(dry_run, clean, delete_outputs)
+    @override
+    def destroy(self, dry_run: bool) -> None:
+        super().destroy(dry_run)
 
-    def clean(self, dry_run: bool, delete_outputs: bool, clear_schemas: bool):
+    @override
+    def clean(self, dry_run: bool, delete_outputs: bool):
         if delete_outputs:
-            # producer clean up job will delete output topics
-            self.clean_app(
+            self.__clean_app(
                 values=self.to_helm_values(),
                 dry_run=dry_run,
                 delete_outputs=delete_outputs,
                 retain_clean_jobs=self.config.retain_clean_jobs,
             )
 
-            if self.to and self.handlers.schema_handler and clear_schemas:
```

Comment:
Actually I think we can remove this whole delete_schemas for KafkaApp. We only need it for KafkaConnect

Reply:
So it should always delete the schema?

Reply:
No, not at all. The streams app clean up does this